packages/material-ui/src/styles/createStyles.js

```diff
@@ -1,3 +1,17 @@
+import warning from 'warning';
+import { createStyles as createStylesOriginal } from '@material-ui/styles';
+
+let warnOnce = false;
+
+// To remove in v5
 export default function createStyles(styles) {
-  return styles;
+  warning(
```

Comment:
Personally I would defer all the deprecations to 4.1. Reduces the amount of churn when switching from 3.9 to 4.0

Reply:
OK, I'm commenting.